### ISSUE TITLE
Multiple glTF models

### ISSUE THREAD

**Issue by m-bob** [moderngl/moderngl-window#153]
Is it possible to load multiple glTF models? If yes, is there an example of doing this?

**Comment by einarf** [comment]
Just look at the basic example : https://github.com/moderngl/moderngl-window/blob/master/examples/cube_model.py

We're drawing an entire scene here, but there's only one cube in the scene. You can of course also load multiple gltf scenes and draw all of them if needed.

Your question is a bit vague, but hopefully that was somewhat useful.

**Comment by m-bob** [comment]
You're right!

I've been playing with the file gltf_scenes.py. Because you've been
insistent (thanks!) I finally was able to see more than one glTF scene at a
time. Here's an example -- which I hope might help someone. I loaded three
scenes:
        self.scene1 = self.load_scene('Buggy/glTF-Binary/Buggy.glb')
        self.scene2 = self.load_scene('SciFiHelmet/glTF/SciFiHelmet.gltf')
        self.scene3 =
self.load_scene('CesiumMilkTruck/glTF/CesiumMilkTruck.gltf')

Then I placed corresponding draw calls in the render routine:
        self.scene1.draw (projection_matrix=self.camera.projection.matrix,
camera_matrix=camera_matrix, time=time,)
        self.scene
2.draw (projection_matrix=self.camera.projection.matrix,
camera_matrix=camera_matrix, time=time,)

        self.scene3.draw
(projection_matrix=self.camera.projection.matrix,
camera_matrix=camera_matrix, time=time,)

Now I can see (pieces of) all three objects at the same time! The composite
scene is ugly, and only pieces of some of the objects appear, but all three
objects are visible, and I assume the issues are because of object and
camera placement. I tried many combinations, and very few combinations are
even slightly compatible -- usually only one object shows up. But this
combination shows multiple objects. When I'm rendering my own objects --
and when I've learned more OpenGL, I expect I can handle the various issues.

Thanks so much!
